src/FC/GPS.cpp

```diff
@@ -0,0 +1,84 @@
+#include "GPS.h"
+#include <Wire.h>
+
+
+namespace GPS {
+
+    SFE_UBLOX_GNSS myGNSS;
+    uint32_t gpsUpdatePeriod = 40 * 1000;
+
+    Comms::Packet latLongPacket = {.id = 11};
```

Comment:
The ID of the gps packet is `6`, according to the [E-1 Design spreadsheet](https://docs.google.com/spreadsheets/d/1xNmjrb5a28FfVg-MG9jUBRiPyfdkjo7HiqPWP41N_IQ/edit#gid=1524498060)